libp2p/Host.h

```diff
@@ -174,10 +173,31 @@ class Host: public Worker
     size_t peerCount() const;
 
     /// Get the address we're listening on currently.
-    std::string listenAddress() const { return m_tcpPublic.address().is_unspecified() ? "0.0.0.0" : m_tcpPublic.address().to_string(); }
+    std::string listenAddress() const
```

Comment:
Why is this change needed?

Reply:
m_tcpPublic won't be initialized if devp2p isn't started, so we should instead return the node table's listen information rather than the host's (m_tcpPublic) in this case.